dev-guide/cluster-version-operator/dev/clusteroperator.md

```diff
@@ -57,8 +56,20 @@ status:
     - name: operator
       # The string "0.0.1-snapshot" is substituted in the manifests when the payload is built
       version: "0.0.1-snapshot" 
+    - name: operator-image
```

Comment:
qq, do we have a list of operators will support multi-arch (in other words, the list of operators which have to add those new entries in their manifest)?

Reply:
Ideally, all cluster operators should do it. We can get the list by `oc get co`.